itests/testdata/reward_distribution_rolllback_testdata.go

```diff
@@ -0,0 +1,101 @@
+package testdata
+
+import (
+	f "github.com/wavesplatform/gowaves/itests/fixtures"
+	utl "github.com/wavesplatform/gowaves/itests/utilities"
+)
+
+func GetRewardDistributionAfterF14Before19TestData(suite *f.BaseSuite, addresses AddressesForDistribution,
+	height uint64) RewardDistributionTestData[RewardDistributionExpectedValues] {
+	return NewRewardDistributionTestData(addresses,
+		RewardDistributionExpectedValues{
+			MinersSumDiffBalance: int64(utl.GetCurrentReward(suite, height)),
+			DaoDiffBalance:       0,
+			XtnDiffBalance:       0,
+			Term:                 utl.GetRewardTermCfg(suite),
+		})
+}
+
+// 2 miners,dao, xtn, initR=600000000, increment = 100000000, desiredR = 800000000
+// ("preactivated_14_supported_19_20/2miners_dao_xtn_without_f20.json")
+// NODE - 858.
+func GetRollbackBeforeF19TestData(suite *f.BaseSuite, addresses AddressesForDistribution,
+	height uint64) RewardDistributionTestData[RewardDistributionExpectedValues] {
+	return NewRewardDistributionTestData(
+		addresses,
+		RewardDistributionExpectedValues{
+			MinersSumDiffBalance: int64(utl.GetCurrentReward(suite, height)) - 2*int64(utl.GetCurrentReward(suite, height))/3,
```

Comment:
HTTP request count will be decreased if you create a new variable with result of `utl.GetCurrentReward(suite, height)` function call and reuse it (fix it in all places).

Reply:
Done